dmscripts/generate_framework_agreement_signature_pages.py

```diff
@@ -129,11 +129,12 @@ def render_html_for_suppliers_awaiting_countersignature(rows, framework, templat
 
         logger.info(f"generating coutersigned agreement PDF for {data['supplier_id']}")
 
-        html_dir = output_dir / str(data['supplier_id'])
+        name_prefix = f"{data['supplier_id']}-{data['agreementId']}"
```

Comment:
am getting `KeyError: 'agreementId'` exception on this line when running locally